web/modules/custom/hoeringsportal_citizen_proposal/src/Form/ProposalFormApprove.php

```diff
@@ -43,6 +43,52 @@ public function buildProposalForm(array $form, FormStateInterface $formState): a
       '#text' => $adminFormStateValues['approve_intro']['value'] ?? '',
     ];
 
+    $form['author'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'div',
+      '#attributes' => [
+        'class' => [
+          'd-flex', 'justify-content-between', 'mt-3', 'border-bottom', 'pb-3',
+        ],
+      ],
+    ];
+
+    $form['author']['name_wrapper'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'div',
+    ];
+
+    $form['author']['name_wrapper']['approve_form_name'] = [
+      '#prefix' => '<h5>' . $this->t('Name') . '</h5>',
+      '#type' => 'processed_text',
+      '#text' => $defaltValues['name'],
+      '#format' => 'filtered_html',
+    ];
+
+    $form['author']['email_wrapper'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'div',
+    ];
+
+    $form['author']['email_wrapper']['approve_form_email'] = [
+      '#prefix' => '<h5>' . $this->t('E-mail') . '</h5>',
+      '#type' => 'processed_text',
+      '#text' => $defaltValues['email'],
+      '#format' => 'filtered_html',
+    ];
+
+    $form['author']['email_display_wrapper'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'div',
```

Comment:
I think using `'#type' => 'container'` is the Drupal way to do this.

Reply:
Fixed